### PR TITLE
[M17]:<title> Tags added to respective pages.

### PR DESCRIPTION
**Pull request by Yashaswini19**
**What's in the PR**
Inclusions of the respective related titles to the web pages using <title> tags in the code.

### CODE REVIEW

File: inception-app-webapp/.temp-inception-app-webapp - INCEpTION-classpath-arg-1570345701574.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...l\9.0.17\tomcat-embed-el-9.0.17.jar;C:\Users\vivek\.m2\repository\org\apache\tomcat\embed\tomcat-embed-websocket\9.0.17\tomcat-embed-websocket-9.0.17.jar;C:\Users\vivek\.m2\repository\org\springframework\boot\spring-boot-autoconfigure\2.1.4.RELEASE\spring-boot-autoconfigure-2.1.4.RELEASE.jar;C:\Users\vivek\.m2\repository\org\springframework\boot\spring-boot-starter-validation\2.1.4.RELEASE\spring-boot-starter-validation-2.1.4.RELEASE.jar;C:\Users\vivek\.m2\repository\javax\validation\validation-api\2.0.1.Final\validation-api-2.0.1.Final.jar;C:\Users\vivek\.m2\repository\com\giffing\wicket\spring\boot\starter\wicket-spring-boot-starter\2.1.7\wicket-spring-boot-starter-2.1.7.jar;C:\Users\vivek\.m2\repository\com\giffing\wicket\spring\boot\starter\wicket-spring-boot-context\2.1.7\wicket-spring-boot-context-2.1.7.jar;C:\Users\vivek\.m2\repository\org\springframework\spring-context-support\5.1.10.RELEASE\spring-context-support-5.1.10.RELEASE.jar;C:\Users\vivek\.m2\repository\org\apache\wicket\wicket-native-websocket-javax\8.5.0\wicket-native-websocket-javax-8.5.0.jar;C:\Users\vivek\.m2\repository\org\apache\wicket\wicket-native-websocket-core\8.5.0\wicket-native-websocket-core-8.5.0.jar
```
Comment (munterkalmsteiner) [inline inception-app-webapp/.temp-inception-app-webapp - INCEpTION-classpath-arg-1570345701574.txt]: Do not submit files to the repository that are not needed.

File: inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalRecommenderTraitsEditor.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 -->
 <html xmlns="http://www.w3.org/1999/xhtml"
   xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.4-strict.dtd">
-<wicket:extend>
```
Comment (munterkalmsteiner) [inline inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalRecommenderTraitsEditor.html]: Why this?

File: inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommenderTraitsEditor.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
 -->
 <html xmlns="http://www.w3.org/1999/xhtml"
   xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.4-strict.dtd">
-<wicket:extend>
```
Comment (munterkalmsteiner) [inline inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/doccat/OpenNlpDoccatRecommenderTraitsEditor.html]: Also here, why? This seems to be unrelated to the improvement you suggest.

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
Looks good, except for some issues I commented.

File: inception-app-webapp/.temp-inception-app-webapp - INCEpTION-classpath-arg-1570345701574.txt (hunk 1 above)
Comment (munterkalmsteiner) [reply inception-app-webapp/.temp-inception-app-webapp - INCEpTION-classpath-arg-1570345701574.txt]: This file still needs to be removed

**Review by munterkalmsteiner — CHANGES_REQUESTED** [review]
Looks good, except for the file that should not be in the repository. Please remove.

**Comment by Yashaswini19** [comment]
File Removed.

**Review by munterkalmsteiner — APPROVED** [review]

**Pull request closed without merging**